doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_partitioning.rst

```diff
@@ -11,6 +11,47 @@ Partitioning device memory is a crucial aspect of managing how a device's storag
 By default, the Partition Manager in the system dynamically generates a partition map, which is suitable for most applications that do not use Device Firmware Upgrades (DFU).
 For scenarios involving DFU, read the following sections.
 
+.. _bootloader_partitioning_partitions_file:
+
+Partition map file
+******************
+
+After you enable the Partition Manager, it will :ref:`start generating <pm_build_system>` the :file:`partitions.yml` file in the build folder directory.
+In this file, you will detailed information about the memory layout used for the build.
+
+The :file:`partitions.yml` file is present also if the Partition Manager generates the partition map dynamically.
+You can use this file as a base for your static partition map.
+
+.. _bootloader_partitioning_partitions_file_report:
+
+Partition reports
+=================
+
+After building the application, you can print a report of how the partitioning has been handled for a bootloader, or combination of bootloaders, by using partition memory reports.
+
+Depending on your development environment, you can use one of the following options:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      Use the extension's `Memory report`_ feature, which shows the size and percentage of memory that each symbol uses on your device for RAM, ROM, and partitions.
+      Click the :guilabel:`Memory report` button in the :guilabel:`Actions View` to generate the report.
```

Comment:
If we want to be even more specific, we can instruct users to then choose the "partitioning" tab.